### PR TITLE
O3-1294:User should be able to navigate back to username screen

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements
- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

#### For changes to apps
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).

#### If applicable
- [x] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

## Summary
A user is able to navigate from the password input screen to the username input screen to modify their username. 

## Screenshots

https://user-images.githubusercontent.com/33891016/171759709-c9280d16-1011-40af-ac41-0f0f354e8ff4.mp4

### CODE REVIEW

**Comment by CynthiaKamau** [comment]
LGTM!

File: packages/apps/esm-login-app/src/login/login.component.tsx [hunk 1]
```diff
@@ -148,6 +149,19 @@ const Login: React.FC<LoginProps> = ({ history, location, isLoginEnabled }) => {
             onClick={() => setErrorMessage("")}
           />
         )}
+        {showPassword ? (
+          <div style={{ width: "23rem" }}>
+            <Button
+              className={styles["back-button"]}
+              kind="ghost"
+              iconDescription="Back"
```
Comment (denniskigen) [inline packages/apps/esm-login-app/src/login/login.component.tsx:157]:
```suggestion
              iconDescription="Back to username"
```

File: packages/apps/esm-login-app/src/styles.scss [hunk 2]
```diff
@@ -115,6 +115,13 @@
   width: 18rem;
 }
 
+.back-button {
+  width: 58px;
+  height: 48px;
+  margin: 133px 426px 0 0;
+  padding: 13px 3px 17px 0;
```
Comment (denniskigen) [inline packages/apps/esm-login-app/src/styles.scss:122]:
```suggestion
  height: 3rem;
```

The `width`, `margin` and `padding` values can be omitted in this case.

**Review by denniskigen — COMMENTED** [review]
Nice work, @jwnasambu. I've left a few suggestions.

File: packages/apps/esm-login-app/src/styles.scss (hunk 2 above)
Comment (jwnasambu) [reply packages/apps/esm-login-app/src/styles.scss:122]: @denniskigen omitting `width`, `margin` and `padding` values  changes the design provided on this link https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/6220c7c1f6a5775abf7846ef. to something like thi
Screenshot 2022-06-06 at 7 54 18 PM
s

File: packages/apps/esm-login-app/src/styles.scss [hunk 3]
```diff
@@ -115,6 +115,17 @@
...
+.back-button {
+  // width: 58px;
+  // height: 48px;
+  // margin: 133px 426px 0 0;
+  // padding: 13px 3px 17px 0;
+  margin-left: -1rem;
+  &:hover {
+    background-color: none;
+  }
+}
```
Comment (denniskigen) [inline packages/apps/esm-login-app/src/styles.scss:127]:
```suggestion
.back-button {
  height: 3rem;
  
  svg {
    margin-right: 0.5rem;
  } 
}
```

Comment (denniskigen) [reply packages/apps/esm-login-app/src/styles.scss:127]: Please delete the commented out styles. Also, you could apply a right margin to the SVG arrow icon as shown above. You should not need to remove the background colour of the button on hover - Carbon's default styles handle that just fine.

**Review by denniskigen — CHANGES_REQUESTED** [review]
Please use a different commit message for fixing up the work.

**Review by denniskigen — APPROVED** [review]

**Pull request merged**